pages/how-it-works.js

```diff
@@ -0,0 +1,57 @@
+import PageWrapper from '../components/common/pageWrapper'
+import Card from '../components/common/card'
+import UnderlinedHeading from '../components/common/underlinedHeading'
+import SubHeading from '../components/common/subheading'
+import TextLink from '../components/common/textLink'
+import { Text, Box, Image } from '@chakra-ui/core'
+
+const HowItWorks = () => (
```

Comment:
not understanding the distinction between HiW and FAQ... seems some of the copy is exactly the same and both are in Q/A format. what is the distinction in your mind / how are you deciding what goes where, and what do you think about cross-linking ?

Reply:
fine with cross linking.

in my head FAQ is more direct Q,A superset of HiW